static/bidder-info/zeta_global_ssp.yaml

```diff
@@ -1,4 +1,4 @@
-endpoint: https://ssp.disqus.com/bid/prebid-server?sid=GET_SID_FROM_ZETA&shortname=GET_SHORTNAME_FROM_ZETA
+endpoint: https://ssp.disqus.com/bid/prebid-server?sid=GET_SID_FROM_ZETA
```

Comment:
@abermanov-zeta  please update json framework tests to use endpoints defined in bidder-info yaml. This ensures that adapter code is compatible with actual endpoint mentioned in yaml.

https://github.com/prebid/prebid-server/blob/754a0ebcdc8b3ac5666986fc81ea762f1b8e1de9/adapters/zeta_global_ssp/zeta_global_ssp_test.go#L12-L16

Reply:
> @abermanov-zeta please update json framework tests to use endpoints defined in bidder-info yaml. This ensures that adapter code is compatible with actual endpoint mentioned in yaml.
> 
> https://github.com/prebid/prebid-server/blob/754a0ebcdc8b3ac5666986fc81ea762f1b8e1de9/adapters/zeta_global_ssp/zeta_global_ssp_test.go#L12-L16

@abermanov-zeta requesting to address comment to move forward with this PR

Reply:
@onkarvhanumante Sorry for the delay, I was on vacation. Tests have been updated. Thank you